docs/DeveloperGuide.adoc

```diff
@@ -415,66 +415,78 @@ See <<UsingTravis#deploying-github-pages, UsingTravis.adoc>> to learn how to dep
 
-*Method 3: Using Gradle (headless)*
+*Method 3: Using Gradle (Headless)*
 
-Thanks to the https://github.com/TestFX/TestFX[TestFX] library we use, our GUI tests can be run in the _headless_ mode. In the headless mode, GUI tests do not show up on the screen. That means the developer can do other things on the Computer while the tests are running.
+* Open a console and run the command `gradlew clean headless allTests` to run tests in headless mode. If you are using Mac/Linux system, run the command `./gradlew clean headless allTests` instead.
 
-To run tests in headless mode, open a console and run the command `gradlew clean headless allTests` (Mac/Linux: `./gradlew clean headless allTests`)
+[NOTE]
+Thanks to the https://github.com/TestFX/TestFX[TestFX] library, our GUI tests can be run in the _headless_ mode. In the headless mode, GUI tests do not show up on the screen. That means the developer can do other things on the computer while the tests are running.
 
 === Types of tests
 
 We have two types of tests:
 
-.  *GUI Tests* - These are tests involving the GUI. They include,
-.. _System Tests_ that test the entire App by simulating user actions on the GUI. These are in the `systemtests` package.
-.. _Unit tests_ that test the individual components. These are in `seedu.ptman.ui` package.
-.  *Non-GUI Tests* - These are tests not involving the GUI. They include,
-..  _Unit tests_ targeting the lowest level methods/classes. +
+.  *GUI Tests*
+* These are tests involving the GUI. They include:
+.. _System tests_ that test the entire App by simulating user actions on the GUI. These are in the `systemtests` package.
+.. _Unit tests_ that test the individual components. These are in the `seedu.ptman.ui` package.
+.  *Non-GUI Tests*
+* These are tests not involving the GUI. They include:
+..  _Unit tests_ targeting methods/classes at the lowest level. +
 e.g. `seedu.ptman.commons.StringUtilTest`
-..  _Integration tests_ that are checking the integration of multiple code units (those code units are assumed to be working). +
+..  _Integration tests_ that are checking the integration of multiple code units. Those code units are assumed to be working. +
 e.g. `seedu.ptman.storage.StorageManagerTest`
-..  Hybrids of unit and integration tests. These test are checking multiple code units as well as how the are connected together. +
+..  Hybrids of _unit_ and _integration tests_. These test are checking multiple code units as well as how they are connected together. +
 e.g. `seedu.ptman.logic.LogicManagerTest`
 
 
 === Troubleshooting Testing
-**Problem: `HelpWindowTest` fails with a `NullPointerException`.**
+
+If you discover any mistakes in the tests, follow this format to troubleshoot testing:
+
+**Problem 1:** `HelpWindowTest` fails with a `NullPointerException`.
 
 * Reason: One of its dependencies, `UserGuide.html` in `src/main/resources/docs` is missing.
 * Solution: Execute Gradle task `processResources`.
 
+**Problem 2:** etc.
```

Comment:
This doesn't add anything useful. I think we should stick with just 1 problem if we have nothing to add.